server/models/Meetup.js

```diff
@@ -0,0 +1,45 @@
+const mongoose = require("mongoose");
+
+const meetupSchema = new mongoose.Schema({
+  location: {
+    type: String,
+  },
+  locationAddress: {
+    type: String,
+  },
+  locationCityStateZip: {
+    type: String,
+  },
+  organizer: {
+    type: mongoose.Schema.Types.ObjectId,
+    ref: "user",
+  },
+  name: {
+    type: String,
+  },
+  date: {
+    type: Date,
+  },
+  timeStart: {
+    type: String,
+  },
+  timeEnd: {
+    type: String,
+  },
+  attendees: [
+    {
+      type: mongoose.Schema.Types.ObjectId,
+      ref: "user",
+    },
+  ],
+  image: {
+    type: String,
+    default:
+      "https://images.unsplash.com/photo-1578729370305-131f65fbd716?ixid=MnwxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHx8&ixlib=rb-1.2.1&auto=format&fit=crop&w=880&q=80",
+  },
+  description: {
+    type: String,
+  },
+});
+
+module.exports = Meetup = mongoose.model("meetup", meetupSchema);
```

Comment:
This `Meetup` is redundant. `module.exports = mongoose.model("Meetup", meetupSchema);`
Mongoose provides an optional `timestamp` field that can be used to automatically manage `createdAt` and `updatedAT`

Reply:
@ezeilo-su I'm not sure I understand what you want me to do here. Do you want me to add the createdAt and updatedAt to the model?

Reply:
also, the linting corrections on the files changed page are not any files that I changed. I think those are from changes merged from main edited/created by another person.

Reply:
> @ezeilo-su I'm not sure I understand what you want me to do here. Do you want me to add the createdAt and updatedAt to the model?

You can add `{ timestamps: true }` to your schema. This will tell mongoose to manage `createdAt` and `updatedAt` automatically.